src/Resources/config/actions.php

```diff
@@ -35,8 +35,10 @@
                 new ReferenceConfigurator('sonata.admin.pool'),
                 new ReferenceConfigurator('twig'),
             ])
+            // NEXT_MAJOR: Remove the alias.
+            ->alias(DashboardAction::class, 'sonata.admin.action.dashboard')
```

Comment:
```suggestion
            ->alias(DashboardAction::class, 'sonata.admin.action.dashboard')
                ->setDeprecated(true)
```
https://symfony.com/doc/4.4/service_container/alias_private.html#deprecating-service-aliases

Reply:
setDeprecated doesn't exist.

I tried something based on the others deprecations.